<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UI_About</class>
 <widget class="QWidget" name="UI_About">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>374</width>
    <height>340</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>374</width>
    <height>340</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>374</width>
    <height>340</height>
   </size>
  </property>
  <property name="focusPolicy">
   <enum>Qt::ClickFocus</enum>
  </property>
  <property name="windowTitle">
   <string>关于本软件…</string>
  </property>
  <widget class="Common_Button" name="btnClose">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>290</y>
     <width>100</width>
     <height>41</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>41</height>
    </size>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>255</x>
     <y>30</y>
     <width>140</width>
     <height>250</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>140</width>
     <height>250</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>140</width>
     <height>250</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>200</width>
     <height>41</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>41</height>
    </size>
   </property>
   <property name="clearButtonEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>220</width>
     <height>220</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>220</width>
     <height>220</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>220</width>
     <height>220</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Common_Button</class>
   <extends>QPushButton</extends>
   <header>state_button.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
